tests/core/backend_connection/test_apiv1_anonymous_cmds.py

```diff
@@ -1,75 +1,13 @@
 # Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
 
 import pytest
-import trio
 
 from parsec.api.protocol import APIV1_ANONYMOUS_CMDS
-from parsec.api.protocol.types import OrganizationID
-from parsec.core.types import BackendOrganizationAddr
-from parsec.core.backend_connection import (
-    BackendNotAvailable,
-    BackendConnectionRefused,
-    apiv1_backend_anonymous_cmds_factory,
-)
+from parsec.core.backend_connection import apiv1_backend_anonymous_cmds_factory
 
-from parsec.backend.backend_events import BackendEvent
 from tests.core.backend_connection.common import ALL_CMDS
 
 
-@pytest.mark.trio
```

Comment:
This is why we should have keep the apiv1 ping untouched: now we no longer test parts of the APIv1.

The impact here is limited given we don't plan on doing any modification in APIv1, but this put us at risk anyway so we should avoid this next time ;-)